dev-docs/modules/categoryTranslation.md

```diff
@@ -13,18 +13,18 @@ sidebarType : 1
 
 {:.no_toc}
 
-This module converts the IAB sub category to FreeWheel industry group identifiers. The FreeWheel identifiers ensure competitve separation of industries and products. 
+This module converts the IAB sub category to Ad server industry group identifiers. The identifiers ensure competitve separation of industries and products. 
 
-Each bid request must return one [IAB subcategory](https://support.aerserv.com/hc/en-us/articles/207148516-List-of-IAB-Categories).
+Each bidder must return one [IAB subcategory](https://support.aerserv.com/hc/en-us/articles/207148516-List-of-IAB-Categories) if they want to ensure creative exclusivity.
```

Comment:
Not necessarily a change but we've been using competitive separation to describe this process, should we switch to creative exclusivity? Is that a FreeWheel only term or industry-wide?

Reply:
Updated to competitive separation.